proto.lock

```diff
@@ -363,6 +363,11 @@
                 "name": "age",
                 "type": "int32"
               },
+              {
+                "id": 101,
+                "name": "newnew",
+                "type": "int32"
+              },
```

Comment:
this indicates a change recorded by a `protolock commit` invocation, after changing the testdata/test.proto as seen below